cmd/config.go

```diff
@@ -197,10 +172,37 @@ type Config struct {
 	SubscriberAgreementURL string
 }
 
+// ServiceConfig contains config items that are common to all our services, to
+// be embedded in other config structs.
+type ServiceConfig struct {
+	// DebugAddr is the address to run the /debug handlers on.
+	DebugAddr string
+	AMQP      *AMQPConfig
+}
+
+// AMQPConfig describes how to connect to AMQP, and how to speak to each of the
+// RPC services we offer via AMQP.
+type AMQPConfig struct {
+	Server            string
+	Insecure          bool
+	RA                *RPCServerConfig
```

Comment:
I gather that the idea here is that each entity has entries here for its depedecies. For example, the `WFE` one would have entries for `RA` and `SA`.  That seems fine for individually configuring timeouts, but I don't think we should put the server names in there.  In addition to being duplicative, you risk having a mismatch between the queue that a component is configured to listen on and the one that its clients are sending messages to.